Kconfig

```diff
@@ -228,6 +228,12 @@ config DEBUG_IPC_COUNTERS
 	help
 	  Select for enabling tracing IPC counter in SRAM_REG mailbox
 
+config DEBUG_ALIGNMENT
+	bool "Power of 2 alignment"
+	default n
+	help
+	  Select to enable compile- and run-time power-of-2 alignment checks
```

Comment:
Alignment where ? can we provide more context.

Reply:
All alignments for aligned memory allocations are checked (or intended to be checked).

Reply:
@lgirdwood help text extended